src/Liquid/StandardFilters.php

```diff
@@ -18,7 +18,7 @@
  */
 class StandardFilters
 {
-	
+
```

Comment:
Can you please revert this change/

Reply:
I can't. PHPStorm auto fixing it before commit.

Reply:
Can you at least add another test please?